File: 2026/06-June/18-Thursday/solution.go
import "math"

func angleClock(hour int, minutes int) float64 {
    hourAngle := float64(hour%12)*30.0 + float64(minutes)*0.5
    minuteAngle := float64(minutes) * 6.0

    diff := math.Abs(hourAngle - minuteAngle)

    return math.Min(diff, 360.0-diff)
}

File: 2026/06-June/18-Thursday/solution.cs
public class Solution {
    public double AngleClock(int hour, int minutes) {
        double hourAngle = (hour % 12) * 30.0 + minutes * 0.5;
        double minuteAngle = minutes * 6.0;

        double diff = Math.Abs(hourAngle - minuteAngle);

        return Math.Min(diff, 360.0 - diff);
    }
}
